src/main/java/dk/bankdata/api/jaxrs/jwt/JwtToken.java

```diff
@@ -6,7 +6,9 @@
  * Contains JWT information.
  */
 public class JwtToken {
-    private final JwtClaims jwtClaims;
+    private JwtClaims jwtClaims;
+
+    protected JwtToken() {}
```

Comment:
you could call the other constructor with `null` then it can still be final, i.e., immutable
```suggestion
    protected JwtToken() {
        this(null);
    }
```